tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -206,46 +202,6 @@ def _check_conversion_params(conversion_params, is_v2=False):
                      "(to disable main graph conversion).")
 
 
-def _check_trt_version_compatibility():
```

Comment:
Let's split this change to a follow up PR. I meant, the change to move this check to the initialization of the package.

Reply:
Fixed